cli/azd/pkg/input/console.go

```diff
@@ -149,24 +151,43 @@ func (c *AskerConsole) MessageUxItem(ctx context.Context, item ux.UxItem) {
 	}
 }
 
+func (c *AskerConsole) spinnerText(title, charset string) string {
+
+	spinnerLen := len(charset) + 1 // adding one for the empty space before the message
+
+	if len(title)+spinnerLen >= c.consoleWidth {
+		postfix := "..."
+		return fmt.Sprintf("%s%s", title[:c.consoleWidth-spinnerLen-len(postfix)], postfix)
+	}
+	return title
+}
+
 func (c *AskerConsole) ShowSpinner(ctx context.Context, title string, format SpinnerUxType) {
 	if c.formatter != nil && c.formatter.Kind() == output.JsonFormat {
 		// Spinner is disabled when using json format.
 		return
 	}
 
+	if c.consoleWidth == cMinConsoleWidth {
+		// no spinner for consoles with width <= cMinConsoleWidth
+		c.Message(ctx, title)
+		return
+	}
```

Comment:
Should this check be `<=` then based on your comment?

Reply:
`c.consoleWidth` is set to `cMinConsoleWidth` wihtin the constructor when the current terminal width is less or equal to the minimum.  But I will update as it is indeed confusing